parsec/backend/user.py

```diff
@@ -343,6 +355,25 @@ async def get_user_with_device(
         """
         raise NotImplementedError()
 
+    async def get_user_from_email(self, organization_id: OrganizationID, user_email: str) -> User:
+        """
```

Comment:
I would add some info regarding the email vs parsec id discussion: 
```suggestion
        """
        Returns the (non-revoked) user associated with the given email address.
```

Reply:
1c9f1322aeaabe5880b1c6abdc5ecb928db70cc1